static/docs/use-cases/shared-storage-on-nfs.md

```diff
@@ -0,0 +1,144 @@
+# Shared Storage on NFS
+
+In the modern software development environment, teams are working together on
+same dataset to get the results. It became necessary that data is accessible and
+every team member has a same updated dataset. For this example, we will be using
```

Comment:
it's better to say that attached NFS storage is used as a shared storage usually ... not "we will ..." ... it's not about us, it's about NFS being widely used for this scenario. Here we are just explaining how DVC can help manage data in this case.

Reply:
Done